### ISSUE TITLE
Migration published every time even though one already exists

### ISSUE THREAD

**Issue by rihardssceredins** [spatie/laravel-schedule-monitor#13]
```
laravel new ohdear
cd ohdear
composer require spatie/laravel-schedule-monitor
composer require ohdearapp/ohdear-php-sdk
php artisan vendor:publish --provider="Spatie\ScheduleMonitor\ScheduleMonitorServiceProvider" --tag="migrations"
php artisan vendor:publish --provider="Spatie\ScheduleMonitor\ScheduleMonitorServiceProvider" --tag="config"
ls -la database/migrations
```

As you'll see `CreateScheduleMonitorTables` migration is published in migrations folder. If you run now `php artisan vendor:publish --all` it will create another migration even though one already exists.

**Comment by mattiasgeniar** [comment]
@rihardssceredins : when do you run the `php artisan vendor:publish --all` command?

you're right, if you run that, it'll publish the database migration again. But it'll do that for about 75% of the open-source packages out there, because of the way this is implemented.

One alternative, is to use the `$this->loadMigrationsFrom()` method in this package instead (see [docs](https://laravel.com/docs/8.x/packages#migrations)), but this has the downside of giving less visiblity to the users. It's a "hidden" migration, I think we all prefer to have the migrations visible in database/migrations instead.

In other words: you're right, this is happening, but I don't think it'll get a fix because of the downsides.

**Comment by rihardssceredins** [comment]
@mattiasgeniar I currently run this command during deployment in my Kubernetes cluster to ensure that all vendor files are published. Reading the documentation now I guess it I should run it locally, commit everything in git and shouldn't have to run it during deployment.

> It's a "hidden" migration, I think we all prefer to have the migrations visible in database/migrations instead.

You're right about this.

<img width="469" alt="Screenshot 2020-09-22 at 14 21 59" src="https://user-images.githubusercontent.com/23099574/93876284-485a3780-fcdf-11ea-9b32-0ed1716602f7.png">

Why Telescope package works correctly and publishes the migration to database/migrations only once if it already is there no matter how many times I run `php artisan vendor:publish --tag=telescope-migrations`? What happens in future if you add another migration? It will duplicate the previous one and the new one?
